README.md

```diff
@@ -1,20 +1,30 @@
-PSP Pendulum smart contract designed to provide access to any currency on top of substrate pendulum chain via chain extension. 
+# Pendulum-Ink-Wrapper
 
-Here is the conversion from primitive type to currency id: https://github.com/pendulum-chain/pendulum/blob/main/runtime/common/src/chain_ext.rs#L124-L143
+A collection of ink contracts that interact with Pendulum's chain extensions.
 
-Access to new currency id requires deployment of new psp pendulum smart contract instance and initializing the contract with correct type_id, code and issuer to support mapping on the chain side.
+## Setup
 
+cargo install cargo-contract --version 2.1.0
 
+## Build
 
-cargo install cargo-contract --version 2.1.0
+`cargo contract build`
 
-Contracts
+## Deploy
 
-If you have cargo-contract (https://github.com/paritytech/cargo-contract) installed, you can easily build with:
+UI to deploy and use: https://contracts-ui.substrate.io/?rpc=ws://127.0.0.1:9944
```

Comment:
Maybe we should change the linked web socket to be one of Foucoco's?
Or we mention multiple web sockets that we 'recommend' in a code box below. I think it's important to link to Foucoco's here.

Reply:
Done 👍